Classes/student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, name = 'Unknown', classroom: true)
+    super(name, age)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Kindly add `parent_permission` parameter to the initialize method with a default value of `true` and also add `parent_permission` to the super call so that we can have access to it from the parent class.

- [ ]  Note that the classroom is not supposed to have a default value, so please remove the default value you added to the classroom.